misc/namespace-tests/buildContainer/pidNamespaceTest.c

```diff
@@ -0,0 +1,41 @@
+ /*
```

Comment:
FFTI:

I'd rather this be in go, but if its more clear to do this in C then I won't complain.

Reply:
Writing this in Go would still require the same system calls. We will keep the C implementation until there is a significant need to port over to Go. We will also get the proper attribution for deriving this from the pause container implementation.